infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -174,6 +181,11 @@
                     + " --version                                                                "
                     + "              Show version\n";
 
+    private static final String MEMORY = "--memory";
```

Comment:
These really don't need to be static constants

Reply:
Shouldn't they be constants? They are used in a lot of places (eg. --address, --memory).
ServerContext, ManagementServer, LogunitServer.

Reply:
I think we should only use constants for numeric values or strings where a value may easily be changed (e.g. timeout) or there is some performance reason.

Here I think it's unlikely that these values will change and it's cumbersome to define these constants in addition to having them in the docopt string.

Maybe if Java supported template literals we could use the constant in the docopt string too...

Reply:
I see. I'll remove them.